src/test/java/homer/common/time/TestDurationConverter.java

```diff
@@ -0,0 +1,23 @@
+package homer.common.time;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.time.Duration;
+
+import org.junit.jupiter.api.Test;
+
+class TestDurationConverter {
+    private static final double DELTA = 0.0001d;
+    private static final long FIVE_HUNDRED = 500;
+
+    @Test
+    void testSeconds() {
+        assertEquals(1, DurationConverter.toSeconds(Duration.ofSeconds(1)), DELTA);
+        assertEquals(0.5, DurationConverter.toSeconds(Duration.ofMillis(FIVE_HUNDRED)), DELTA);
+    }
+
+    void testHours() {
```

Comment:
```suggestion

   @Test
    void testHours() {
```

Reply:
can't format properly on gh so dont commit directly

Reply:
ok, sure. Don't know how i missed that, sorry!